man/fwrite.Rd

```diff
@@ -52,6 +53,7 @@ fwrite(x, file = "", append = FALSE, quote = "auto",
   \item{buffMB}{The buffer size (MB) per thread in the range 1 to 1024, default 8MB. Experiment to see what works best for your data on your hardware.}
   \item{nThread}{The number of threads to use. Experiment to see what works best for your data on your hardware.}
   \item{showProgress}{ Display a progress meter on the console? Ignored when \code{file==""}. }
+  \item{compress}{If compress = \code{"gzip"} or if \code{file} ends in \code{.gz}, even if compress = \code{"none"}, then the output format is gzipped csv. Output to console is never gzipped even if compress = \code{"gzip"}. By default, compress = \code{"none"}.}
```

Comment:
Well documented behaviour. I would put "compress" into "code" tag as it refers to argument variable, together with "=" sign.

Reply:
OK. I reformat documentation like this and describe new `default` option. See https://github.com/Rdatatable/data.table/pull/3278/commits/b3c2ae9fe8d0e8fd5d382bc44f580630e1b3cbb3